### ISSUE TITLE
Install - Absolute Paths

### ISSUE THREAD

**Issue by mirage335** [atom/apm#795]
tldr: Absolute paths must never be part of any permanently written software configuration.

### Prerequisites

* [X ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description
Running "apm install" in a module directory, results in absolute paths, written to "package.json" files, under "node-modules" directory.

Breaks 'portable' uses for "ATOM_HOME" directories, like this.
https://github.com/mirage335/ubiquitous_bash/blob/master/shortcuts/dev/app/devatom.sh#L38

Especially unhelpful for developing specialized IDE environments which may need to use modules (eg. atom-dbg-gdb) directly from git repositories, as this does.
https://github.com/mirage335/arduinoUbiquitous/blob/master/_prog/core.sh#L587

### Steps to Reproduce
1. cd <module directory>
2. apm install
3. grep -r "/home/$USER"
4. echo $?
Last command returns '0', should return '1'.

### Versions

Debian Stable, aka. Debian Stretch
```
apm --version
apm  1.19.0
npm  3.10.10
node 6.9.5 x64
atom 1.27.2
python 2.7.13
git 2.11.0
```
```
atom --version
Atom    : 1.27.2
Electron: 1.7.15
Chrome  : 58.0.3029.110
Node    : 7.9.0
```

### Additional Information

Nope.
